5-Heroku/client/src/todo_api.js

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+const TODOS_ENDPOINT = "todos";
+
+function createItemData(item) {
+  return axios
+    .post(`/${TODOS_ENDPOINT}`, item)
+    .catch(error => error);
```

Comment:
what does this line do?

Reply:
Added now catch on client side with logs & alerts (like you suggested for getItems())

Reply:
this line doesn't do anything...